Library/ViewModels/LoginViewModelTests.swift

```diff
@@ -314,6 +314,7 @@ final class LoginViewModelTests: TestCase {
   func testShowPassword() {
     self.vm.inputs.viewWillAppear()
     self.vm.inputs.viewDidLoad()
+    self.vm.inputs.traitCollectionApplied()
```

Comment:
After you make the previous change I suggested then you won't need this in this particular test because we will have changed the `combineLatest` to a `merge`.

Please add an additional test which tests the `traitCollectionDidChange` input.